src/Configuration/test/Kubernetes.ServiceBinding.Test/PostProcessorsTest.cs

```diff
@@ -495,7 +488,7 @@ public void MongoDbTest_BindingTypeDisabled()
         postProcessor.PostProcessConfiguration(provider, configurationData);
 
         string keyPrefix = GetOutputKeyPrefix(TestBindingName, MongoDbPostProcessor.BindingType);
-        configurationData.Should().NotContainKey($"{keyPrefix}:authenticationDatabase");
+        configurationData.Should().NotContainKey($"{keyPrefix}:username");
```

Comment:
Why are we expecting that `username` should _not_ be included in the configuration data? I thought I read a comment that would offer an explanation (something about avoiding dealing with precedence rules for properties coming from appsettings?), but now I can't find it again

Reply:
This basic unit test asserts that the post-processor does not run when it is explicitly disabled. It is disabled because of the `false` argument in `PostProcessorConfigurationProvider provider = GetConfigurationProvider(postProcessor, MongoDbPostProcessor.BindingType, false);`. In contrast, the test below passes `true` and asserts that the post-processor does run.